test/envoye2e/basic_auth/basic_auth_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"fmt"
+	"istio.io/proxy/test/envoye2e"
+	"istio.io/proxy/test/envoye2e/driver"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+	"time"
+
+	"istio.io/proxy/test/envoye2e/env"
```

Comment:
The import here does not look right to me, should be something like:
```suggestion
	"fmt"
	"os"
	"path/filepath"
	"strconv"
	"testing"
	"time"

	"istio.io/proxy/test/envoye2e"
	"istio.io/proxy/test/envoye2e/driver"
	"istio.io/proxy/test/envoye2e/env"
```

Wondering why CI is not catching it.

Reply:
okok changed.